### DISCUSSION TITLE
Outline Client custom branding?

### DISCUSSION THREAD

**Discussion by SiteSplat** [OutlineFoundation/outline-sdk#78] [General]
Any plans for the outline client to have options to ship it with custom branding?
Something as simple as a color scheme and logo.

**Comment by maddyhof** [comment]
Thanks for reaching out! Yes this request has indeed come up a few times and is on our radar, but to be honest I don't yet have clarity on when it'll be prioritized. Are you able to share more about why you're interested in this capability, to inform our plans?

**Comment by fortuna** [comment]
Moved to https://github.com/Jigsaw-Code/outline-client/issues/1759
